src/Compilers/CSharp/Portable/Binder/MethodGroupResolution.cs

```diff
@@ -85,6 +85,12 @@ public bool IsExtensionMethodGroup
             get { return (this.MethodGroup != null) && this.MethodGroup.IsExtensionMethodGroup; }
         }
 
+        public bool IsExtensionMember(out Symbol extensionMember)
+        {
+            extensionMember = OtherSymbol;
```

Comment:
>OtherSymbol [](http://example.com/codeflow?start=30&length=11)

Is this guaranteed to be null when condition below is false? Could we assert that?

Reply:
The expression below (`ResultKind == LookupResultKind.Viable && MethodGroup is null`) means we're dealing with an extension member.
If we're not dealing with an extension member, we may either have a method group or an other symbol, depending on the case (based on pre-existing logic).